Workspace/src/twist_mux/.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
We really shouldn't have individual .gitignores...

`twist_mux` can come in for now but I think it's going to be rewritten